tool/tctl/common/recordings_command.go

```diff
@@ -91,5 +92,8 @@ func (c *RecordingsCommand) ListRecordings(ctx context.Context, tc auth.ClientI)
 	if err != nil {
 		return trace.Errorf("getting session events: %v", err)
 	}
+	if recordings == nil {
+		recordings = []events.AuditEvent{}
+	}
```

Comment:
Can we remove this? `ShowSessions` should call the shared impls that already handle nil slices.